week3/homework/src/removeToDo.js

```diff
@@ -0,0 +1,18 @@
+const requireActions = require('./actions');
+function removeToDo(req, res) {
+  if (requireActions.getItem(req.params.id)) {
+    res.status(200);
+    res.json(
+      (() => {
+        requireActions.removeToDo(req.params.id);
+        return requireActions.get();
+      })()
+    );
```

Comment:
You could use the same short-circuit you used in the previous functions, for consistency
    
    res.json(requireActions.removeToDo(req.params.id) && requireActions.get());